docs/guides/upgrade-guides/docker.md

```diff
@@ -74,6 +74,8 @@ cd /opt/configuration/environments/manager
 ANSIBLE_ASK_VAULT_PASS=True ./run.sh docker
 ```
 
+Restart the docker service on the manager afterwards manually.
```

Comment:
I think that's not necessary. If required the ./run.sh docker will restart the service.

Reply:
Even better!